contracts/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@confluxfans/contracts",
+  "description": "Common useful contracts in Conflux DAPP developments",
+  "version": "0.1.1",
+  "files": [
+    "**/*.sol",
+    "/build/contracts/*.json",
+    "!/mocks/**/*"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/conflux-fans/conflux-contracts.git"
+  },
+  "keywords": [
+    "solidity",
+    "smart",
+    "contracts",
+    "security"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/conflux-fans/conflux-contracts/issues"
+  },
+  "homepage": "https://github.com/conflux-fans/conflux-contracts",
+  "dependencies": {
+    "@openzeppelin/contracts": "4.1.0"
```

Comment:
hardhat dependency missed?

Reply:
This is contract's package.json, the outside one still have hardhat dependency